### PR TITLE
Use istanbul coverage tool int the webpack coverage

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -99,6 +99,8 @@ jobs:
     - make lint
     - make check-examples-webpack
     - if [ ${IS_HOME} = TRUE ]; then JOB=publish-examples buildtools/deploy.sh; fi
+    after_success:
```
Comment (fredj) [inline .travis.yml]: there's the same code at line 80, is is correct ?

File: .travis.yml [hunk 2]
```diff
@@ -99,6 +99,8 @@ jobs:
...
+    - cat .build/coverage/lcov.info | node ./node_modules/coveralls/bin/coveralls.js
```
Comment (fredj) [inline .travis.yml]: There's the same code line 80, is it correct?

Comment (sbrunner) [reply .travis.yml]: Good question defectively, should I comment this line and we will uncomment it when we move on webpack?

Comment (fredj) [reply .travis.yml]: let's keep it

**Pull request merged**